testsuite/MDAnalysisTests/lib/test_pkdtree.py

```diff
@@ -92,102 +93,98 @@ def test_set_coords():
 boxes_2 = ([10, 10, 10, 90, 90, 90],  # ortho
           [10, 10, 10, 45, 60, 90])  # tri_box
 
-# Find images for a given query vector, here in fractional coordinates
-queries = ([0.5, 0.5, 0.5],  # case box center
```

Comment:
Why did you remove a test?

Reply:
I only removed it because it sits in the middle of the cell, so it does not produce any image. Maybe I should include it just for the sake of proving the negative?

Reply:
jup. it doesn't hurt and is a very common edge case ;)